src/compute-types/src/dyncfgs.rs

```diff
@@ -132,6 +132,14 @@ pub const COMPUTE_REPLICA_EXPIRATION_OFFSET: Config<Duration> = Config::new(
     "The expiration time offset for replicas. Zero disables expiration.",
 );
 
+/// Main switch to enable or disable replica expiration. A false value disables
+/// expiration.
+pub const COMPUTE_REPLICA_EXPIRATION_MAIN_SWITCH: Config<bool> = Config::new(
+    "compute_replica_expiration_main_switch",
```

Comment:
We usually call these feature flags `enable_*`. Any reason you don't like that here? If not I'd prefer using the usual convention because it makes it easier to find the flag. (For example, the flag to enable RTR is called `allow_real_time_recency` which annoyed me a lot!)

Reply:
No good reason! Copy-paste...